### PR TITLE
Pool 21 portfolio sell button tooltips

### PR DESCRIPTION
**Pull request by rogue-hacker**
- Sell button added to Portfolio page token rows
- Add tooltips to Stake, Sell, Burn and Flip action buttons

### CODE REVIEW

File: archetypes/Portfolio/ClaimedTokensTable/ClaimedTokenRow.tsx [hunk 1]
```diff
@@ -51,25 +53,39 @@ export const ClaimedTokenRow: React.FC<ClaimedTokenRowProps & ClaimedRowActions>
...
+                <PortfolioBurnTooltip>
+                    <ActionsButton
+                        size="xs"
+                        variant="primary-light"
+                        disabled={!balance.toNumber()}
+                        onClick={() => onClickCommitAction(poolAddress, side, CommitActionEnum.burn)}
+                    >
+                        Burn
+                    </ActionsButton>
+                </PortfolioBurnTooltip>
```
Comment (chrdevmar) [inline archetypes/Portfolio/ClaimedTokensTable/ClaimedTokenRow.tsx]: this currently renders the placeholder for {time}, it needs to pass the actual expected time in

File: archetypes/Portfolio/ClaimedTokensTable/ClaimedTokenRow.tsx [hunk 2]
```diff
@@ -51,25 +53,39 @@ export const ClaimedTokenRow: React.FC<ClaimedTokenRowProps & ClaimedRowActions>
...
+                    >
+                        {shouldStake ? 'Stake' : 'Unstake'}
+                    </ActionsButton>
+                </PortfolioStakeTooltip>
+                <PortfolioSellTooltip>
+                    <ActionsButton
+                        size="xs"
+                        variant="primary-light"
+                        disabled={!balance.toNumber()}
+                        onClick={() => open(constructBalancerLink(address, NETWORKS.ARBITRUM, true), '_blank')}
```
Comment (chrdevmar) [inline archetypes/Portfolio/ClaimedTokensTable/ClaimedTokenRow.tsx]: this should be selling so need to pass false for `isBuy` parameter

```suggestion
                        onClick={() => open(constructBalancerLink(address, NETWORKS.ARBITRUM, false), '_blank')}
```

**Review by chrdevmar — CHANGES_REQUESTED** [review]

File: archetypes/Portfolio/ClaimedTokensTable/ClaimedTokenRow.tsx (hunk 1 above)
Comment (rogue-hacker) [reply archetypes/Portfolio/ClaimedTokensTable/ClaimedTokenRow.tsx]: Resolved

File: archetypes/Portfolio/ClaimedTokensTable/ClaimedTokenRow.tsx [hunk 3]
```diff
@@ -51,25 +55,46 @@ export const ClaimedTokenRow: React.FC<ClaimedTokenRowProps & ClaimedRowActions>
...
+                    <ActionsButton
+                        size="xs"
+                        variant="primary-light"
+                        disabled={!balance.toNumber()}
+                        onClick={() => open(constructBalancerLink(address, NETWORKS.ARBITRUM, false), '_blank')}
+                    >
+                        Sell
+                    </ActionsButton>
+                </PortfolioSellTooltip>
+                {console.log(expectedExecution)}
```
Comment (chrdevmar) [inline archetypes/Portfolio/ClaimedTokensTable/ClaimedTokenRow.tsx]: remove this

**Review by chrdevmar — CHANGES_REQUESTED** [review]

**Review by chrdevmar — APPROVED** [review]
lgtm

**Pull request merged**